package.json

```
@@ -31,7 +31,13 @@
     "prebuild": "node generate-netlify-env.js",
     "build": "next build",
     "start": "next start",
-    "test": "jest",
+    "test": "npm run test:jest && npm run test:e2e",
```

I think  `e2e` should be its on script and separate from the `npm run test` command. Is this not picked up automatically by the actions to run E2E tests?

```suggestion
    "test": "npm run test:jest && npm run test:e2e",
```

---

https://github.com/open-sauced/open-sauced/blob/main/.github/workflows/development.yml
only `npm run test` is executed in the CI.

---

I see. I associate running `npm run test` with unit tests. Not a blocker for this PR though